src/Spec2-Code/SpCodeInteractionModel.class.st

```diff
@@ -66,23 +66,23 @@ SpCodeInteractionModel >> isScripting [
 SpCodeInteractionModel >> notify: message at: location in: code [
 	| currentSelection stripMessage |
 
-	"self owner withAdapterDo: [ :anAdapter | 
+	"self owner withAdapterDo: [ :anAdapter |
 		anAdapter notify: message at: location in: code ]"
 
-	stripMessage := (message endsWith: '->')
+	stripMessage := (message endsWith: ' ->')
 		ifTrue: [ message allButLast: 3 ]
```

Comment:
I wonder if the right change is not `message allButLast: 2` but I do not know what kind of messages this method expect

Reply:
.. or `(allButLast: 2) trimmed` ?

Reply:
There are only two real senders, one ending its messages with `' ->'` and the without them, so it does not change that much. It's not that important, but the bad math feels just bad (and did hurt me when I used only '->' as message by mistake).